packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx

```diff
@@ -98,21 +103,46 @@ class DynamicModuleLoaderImpl extends React.Component<
                     "Store could not be resolved from React context"
                 );
             }
+        } else {
+            // We will add modules dynamically and due to github issue https://github.com/Microsoft/redux-dynamic-modules/issues/27#issuecomment-464905893
+            // The very first render will not get latest state, to fix that we will need to get latest state from store directly on first render
+            this._getLatestState = ReactReduxContext;
         }
 
         this._addedModules = store.addModules(modules);
     }
 
+    private _wrappedRender = () => {

```

Comment:
Again, something to denote that this is the RR6 path?

Reply:
Sure will rename